.gitmodules

```diff
@@ -92,3 +92,6 @@
 [submodule "vlsi/hammer-mentor-plugins"]
 	path = vlsi/hammer-mentor-plugins
 	url = git@github.com:ucb-bar/hammer-mentor-plugins.git
+[submodule "toolchains/qemu"]
```

Comment:
Probably want to write a doc on this... but we can just make this an issue and do this later.

Reply:
What's there to document? It's installed by the build-toolchains script and is needed the same as other toolchain stuff (e.g. spike). I suppose mentioning that we override any system qemu they have already installed wouldn't hurt.